docs/core/testing/unit-testing-with-mstest.md

```diff
@@ -17,14 +17,16 @@ This tutorial takes you through an interactive experience building a sample solu
 
 ### Creating the source project
```

Comment:
The title and description should mention C#, and the description can be expanded